### PR TITLE
Adding show command (useful for containers with same name) to see ins…

### PR DESCRIPTION
**Pull request by remyd1**
Hi,

This pull request allows a user to use the "show" command which is simply looking in the container directory to display the content of summary, installed, build_..., (...) files.

For instance, it can be useful if you have many containers with the same name and you want to display specific informations about a container from UUID.

Best regards,

Rémy

### CODE REVIEW

File: libexec/cli/show.exec [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+export SAPPNAME SINGULARITY_COMMAND CONTAINERPATH
+shift
+
+message 2 "Showing '$SAPPNAME'\n"
+
+RETVAL=1
+
+NAME="undefined"
+SUMMARY="undefined"
+SIZE="0"
```
Comment (remyd1) [inline libexec/cli/show.exec:69]: I forget to add
NB_FILES=0
here...

**Comment by gmkurtzer** [comment]
I'll check this out as soon as I get to a real computer. Thanks!

Sent from my iPhone

**Comment by gmkurtzer** [comment]
I really like this feature, thanks!

**Comment by remyd1** [comment]
You are welcome. It is useful to me too. I did not find any checksum on the
whole container. So I also thought that a hash/checksum (e.g. md5 or
sha1...) on the whole container could be useful. It would be display in the
'show' command then.
If you plan to make a registry for the sapp applications this feature would
also be important.

Best,
Remy.
Le 26 avr. 2016 20:52, "Gregory M. Kurtzer" notifications@github.com a
écrit :

**Pull request merged**